topics/introduction/slides/index.html

```diff
@@ -93,30 +99,35 @@
 - how to generate a command to execute the tool with the specified inputs and parameters
 - the output datasets the tool should produce and their datatypes
 
+Tools can be viewed as tiny LEGO pieces: each one solves a specific problem, and each can be combined to build complex data analysis.
```

Comment:
"each can be combined to build complex data analysis" -> "they can be combined together to build complex analysis pipelines"

Reply:
clearer, thanks